float32 fore
float32 mid
float32 hind
float32 all_force
bool stance_flg